iDiff/utils/image_utils.go

```diff
@@ -72,3 +168,18 @@ func copyToFile(outfile string, r io.Reader) error {
 
 	return nil
 }
+
+func checkImageID(image string) bool {
+	pattern := regexp.MustCompile("[a-z|0-9]{12}")
+	if exp := pattern.FindString(image); exp != image {
+		return false
+	}
+	return true
+}
+
+func checkImageTar(image string) bool {
```

Comment:
Looking again it might make the most sense to just make these exported functions so they can be called elsewhere through the utils package. I can definitely imagine both of these functions being used in differs.go to handle function calling based on argument types or in argument verification of the command.